### PR TITLE
fix the title of the operations.html

### PR DESCRIPTION
**Pull request by caesarxuchao**
@RichieEscarez you raised an issue on the title of docs/api-reference/operations.html is "paths" rather than "operations". ([here](https://htmlpreview.github.io/?https://github.com/kubernetes/kubernetes/blob/master/docs/api-reference/operations.html))

I couldn't find the exact issue number. Anyway, this PR fixes that, and I have pushed a v1.1 image to gcr.io.

@nikhiljindal @RichieEscarez

### CODE REVIEW

**Comment by nikhiljindal** [comment]
You will have to update other documentation as well where we reference operations.html like: https://github.com/kubernetes/kubernetes/blob/master/hack/gen-swagger-doc/README.md

**Comment by caesarxuchao** [comment]
@nikhiljindal, I'm changing the title in the html, not the file name, so we don't need to change other documentation. Make sense? :)

File: hack/gen-swagger-doc/run-gen-swagger-docs.sh [hunk 1]
```diff
@@ -20,5 +20,4 @@ if [ "$#" -lt 1 ]; then
 fi
 OUTPUT=${2:-${PWD}}
 
-docker run -v ${OUTPUT}:/output gcr.io/google_containers/gen-swagger-docs:v1 https://raw.githubusercontent.com/kubernetes/kubernetes/master/api/swagger-spec/$1.json https://raw.githubusercontent.com/kubernetes/kubernetes/master/pkg/api/$1/register.go
-
+docker run -v ${OUTPUT}:/output gcr.io/google_containers/gen-swagger-docs:v1.1 https://raw.githubusercontent.com/kubernetes/kubernetes/master/api/swagger-spec/$1.json https://raw.githubusercontent.com/kubernetes/kubernetes/master/pkg/api/$1/register.go
```
Comment (nikhiljindal) [inline hack/gen-swagger-doc/run-gen-swagger-docs.sh]: Why v1.1 and not v2?
This is not related in any way to kubernetes release versions right?

**Comment by nikhiljindal** [comment]
aah ok. I will let @RichieEscarez have a look as well.

File: hack/gen-swagger-doc/run-gen-swagger-docs.sh (hunk 1 above)
Comment (caesarxuchao) [inline hack/gen-swagger-doc/run-gen-swagger-docs.sh]: No.

I thought it's just a minor change so I named it v1.1. Is the version important?

Comment (caesarxuchao) [inline hack/gen-swagger-doc/run-gen-swagger-docs.sh]: I'd rather not spend another 10 minutes to build and upload the image if the version doesn't matter so much :)

Comment (nikhiljindal) [inline hack/gen-swagger-doc/run-gen-swagger-docs.sh]: yup. fine as is.
Thanks!

**Comment by RichieEscarez** [comment]
Thanks @caesarxuchao, LGTM

Just to clarify, our goal is to ensure the title of the file that is viewed from in the master branch is the same as the one in gh-pages (ie that shows on kubernetes.io: http://kubernetes.io/v1.0/docs/api-reference/operations.html).

Sounds like @nikhiljindal is okay with the scripts so im adding the LGTM label.

**Comment by caesarxuchao** [comment]
thank you @RichieEscarez!
